packages/@react-facet/core/src/createFacetContext.spec.tsx

```diff
@@ -120,10 +122,10 @@ describe('createFacetContext initial facet', () => {
     consoleLogMock.mockClear()
 
     process.env.NODE_ENV = 'production'
-
     staticFacet.observe(update)
     expect(update).toHaveBeenCalledTimes(1)
     expect(update).toHaveBeenCalledWith(defaultValue)
     expect(consoleLogMock).toHaveBeenCalledTimes(0)
+    process.env.NODE_ENV = 'test'
```

Comment:
This test mutates the outside environment. It could make subsequent tests take different code paths. I restored the `NODE_ENV` to test